search/search_engine_base.py

```diff
@@ -58,5 +59,23 @@ def get_search_engine(index=None):
         """
         Returns the desired implementor (defined in settings).
         """
+
+        # .. toggle_name: default_elastic_search
+        # .. toggle_implementation: WaffleSwitch
+        # .. toggle_default: False
+        # .. toggle_description: This flag forces the use of ElasticSearch.
+        #      It prevents errors from switching to OpenSearch before roll out.
+        # .. toggle_use_cases: temporary
+        # .. toggle_creation_date: 2022-7-11
+        # .. toggle_target_removal_date: None
+        # .. toggle_tickets: TNL-9899
+        # .. toggle_warnings: This temporary feature toggle does not have a target removal date.
+        DEFAULT_ELASTIC_SEARCH_SWITCH = WaffleSwitch('default_elastic_search', __name__)
+
         search_engine_class = _load_class(getattr(settings, "SEARCH_ENGINE", None), None)
-        return search_engine_class(index=index) if search_engine_class else None
+        if search_engine_class:
+            return search_engine_class(index=index)
+        # TNL-9899
```

Comment:
I believe this TNL-9899 logic should precede the `_load_class()` fetch and return to override existing behavior.